back-end/services/AuthService.js

```diff
@@ -24,11 +24,14 @@ const login = async (data) => {
   const email = data.email;
   const password = data.password;
 
-  const user = await findByEmail(email);
-  if (!user) {
+  const userWithoutPassword = await findByEmail(email);
```

Comment:
You don't need to search user twice, you just could find user like `const user = await User.findOne({ email });`

Reply:
fix it